index.js

```diff
@@ -587,27 +582,6 @@ RedisClient.prototype.return_error = function (err) {
     }
 };
 
-// if a callback throws an exception, re-throw it on a new stack so the parser can keep going.
-// if a domain is active, emit the error on the domain, which will serve the same function.
-// put this try/catch in its own function because V8 doesn't optimize this well yet.
-function try_callback(callback, reply) {
-    try {
-        callback(null, reply);
-    } catch (err) {
-        if (process.domain) {
-            var currDomain = process.domain;
-            currDomain.emit('error', err);
-            if (process.domain === currDomain) {
-                currDomain.exit();
-            }
-        } else {
-            process.nextTick(function () {
-                throw err;
-            });
-        }
-    }
-}
-
```

Comment:
Hm, wrapping the user callback into a try catch should not be needed at all - this is actually causing [problems for users] (https://github.com/NodeRedis/node_redis/issues/453). So any user should still be able to handle the errors just as they want and it should be safe to remove this.